python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1451,6 +1452,10 @@ def get_asset_records(
     ) -> Iterable["AssetRecord"]:
         return self._event_storage.get_asset_records(asset_keys)
 
+    @traced
+    def get_asset_event_tags(self, asset_key: AssetKey) -> Sequence[Tuple[str, AbstractSet[str]]]:
```

Comment:
same comment as the cloud version: consider renaming to `get_all_event_tags_for_asset`?